### ISSUE TITLE
FALCON running task://localhost/d_0007_raw_reads failed with Ecoli data

### ISSUE THREAD

**Issue by svm-zhang** [PacificBiosciences/FALCON#451]
Hello Falcon developers,

I have been trying to assemble the Ecoli pacbio data from Kim et al Science data 2014 (SRX669475) using Falcon. I used SMRTanalysis pipeline to preprocess the raw reads and got a set of filtered subreads. I used the following config file for running FALCON:

[ecoli.run2.txt](https://github.com/PacificBiosciences/FALCON/files/516905/ecoli.run2.txt)

The job exited because of d_0007 job failed with exit code=256. And I found this error is quite similar to what has reported in [issue#445](https://github.com/PacificBiosciences/FALCON/issues/445). So I changed -s option for daligner from 1000 to 100 with all other settings unchanged, and it worked. Then I tried to update the daligner submodule by running git submodule update --init, and reinstall FALCON-integrate. The following is what git submodule gives me:

```
a92a5187ce3bdcdcdec346ec828e066314d443bb DALIGNER (remotes/origin/smrtanalysis-3.0.4-39-ga92a518)
d50edb0e741eac74faa5080e981f8849e02265f5 DAMASKER (d50edb0)
d3423119873574171767fb0d0fb40df41efc3f81 DAZZ_DB (remotes/origin/gm-master-35-gd342311)
54779b25532a4e1ace230013fe25f5c51a43ddaa FALCON (0.2JASM-487-g54779b2)
8d4a11c0574c6798ebefe18ade055c63ee402592 FALCON-examples (heads/master)
5cd7666d116e661fc6481d07d70ec01a47bd27d0 FALCON-make (heads/master)
483836854e1bafb6e40cf17f3a9382efe351daad git-sym (heads/master)
aa5d4b65d46ef566d13fae26f1fa2c314882cfd8 pypeFLOW (v0.1.0-112-gaa5d4b6)
```
I re-ran FALCON with the config file I showed upstairs and still got the same type of error. Could you please give me some insights about this error and how can I fix it?

Thanks in advance for any help!

Simo

**Comment by pb-jchin** [comment]
@svm-zhang first, please check the binary of LACheck is actually updated.  Then, run it again, if the problem persists, we will need a sample of the data so we might be able to use it for debugging. In the mean time, I think it is totally ok to use `-s100` .

**Comment by svm-zhang** [comment]
Hello Jason,

Thanks for the quick response. I will try that and let you know the outcome.

Simo

**Comment by svm-zhang** [comment]
Hello Jason,

I manually reinstalled the DALIGNER and made sure LACheck is the latest version. The error still persisted. So I further looked at the log of the job that gave the error, and found that the error was because of the duplicated overlaps. I then found out [this ](https://github.com/thegenemyers/DALIGNER/issues/42), which makes sense. In the form of a(ab, ae) x b(bb, be), could you please give me some sense about what duplicate overlaps mean? I totally understand if you think I should post this question on DALIGNER webpage.

My current solution to get FALCON pass through this stage is to set `skip_check = True` in the config file. But I am not sure if there could be a risk to do so, as some overlaps for some reason should not be used in downstream analyses. For the duplicate overlaps I found for my case,  [this](https://github.com/PacificBiosciences/FALCON/issues/419) thread suggests they won't affect FALCON. 

A humble suggestion is that when the duplicate overlaps occur, could FALCON (the future version) continue to go on the next stage, rather than kill it? In this regard, FALCON still validates the overlaps (compared to completely skip check), and only "OK" and "Duplicate overlaps" are allowed results from LACheck.

And as you suggested earlier, I can always use -s100 at the cost of a little increase in the .las file size.

Please feel free to close this ticket. Thanks for the help!

Simo

**Comment by svm-zhang** [comment]
And if possible could you please refer a SHA1 of DALIGNER with gene's fix for this problem?

**Comment by pb-cdunn** [comment]
I think Gene fixed this in thegenemyers/DALIGNER@0430011a0fd42f940fe59086dc712e9500a0c6f2, June 28. We incorporated that change into our DALIGNER fork pretty quickly. If you still have a similar problem, and if you could provide Gene a test-case, he might be interested.

The setting to suppress **LAcheck**, as you've found, is:

    skip_checks = true

I don't think we can make that the default, since there are other problems (network-related) which we must detect.
